src/frontend/src/utils/urls.ts

```diff
@@ -135,11 +134,15 @@ const bomCreateUnit = () => `${bomEndpoints()}/units/create`;
 /************** Design Review Endpoints *******************************/
 const designReviews = () => `${API_URL}/design-reviews`;
 const designReviewsCreate = () => `${designReviews()}/create`;
+const teamTypes = () => `${designReviews()}/teamType/all`;
```

Comment:
This does fit with design reviews, but we have the corresponding route in the teams file so it probably makes more sense to keep it where it is imo
Why was this moved ?

Reply:
I’m pretty sure this was coming from the merge with develop

Reply:
agree with Sean, it should defo be with the teams stuff